edgedb-protocol/Cargo.toml

```diff
@@ -24,12 +24,13 @@ bitflags = "2.4.0"
 serde = {version="1.0.190", optional=true}
 
 [features]
-default = []
+default = ["macros"]
 with-num-bigint = ["num-bigint", "num-traits"]
 with-bigdecimal = ["bigdecimal", "num-bigint", "num-traits"]
 with-chrono = ["chrono"]
 all-types = ["with-num-bigint", "with-bigdecimal", "with-chrono"]
 with-serde = ["serde"]
+macros = []
```

Comment:
We don't need a feature for macros, since they are not procedural macros and don't require to be in a separate crate.

AFIK, these features for macros are for when crates depend on heavy proc_macro crates that need to be compiled before this crate, which obviously has compilation performance and security implications.